File: articles/algorithms-in-finance-markets-and-risk/java/ExpectedLoss.java
public class ExpectedLoss {
    public static void main(String[] args) {
        double pd = 0.035;
        double lgd = 0.45;
        double ead = 100000.0;
        double expectedLoss = pd * lgd * ead;
        System.out.printf("expected_loss=%.4f%n", expectedLoss);
    }
}

File: articles/algorithms-in-finance-markets-and-risk/rust/expected_loss.rs
fn main() {
    let pd = 0.035_f64;
    let lgd = 0.45_f64;
    let ead = 100000.0_f64;
    let expected_loss = pd * lgd * ead;
    println!("expected_loss={:.4}", expected_loss);
}
